silverCrawler/silverCrawler-war/src/main/java/org/silverpeas/components/silvercrawler/control/SilverCrawlerSessionController.java

```diff
@@ -465,11 +460,10 @@ public boolean checkRWSettingsAccess(boolean throwException)
       throws SilverCrawlerForbiddenActionException {
 
     // First checks read/Write access is enable in silverpeas platform
-    boolean readWriteActivatedInPlatform = getSettings().getBoolean("readWriteActivated", false);
+    boolean readWriteActivatedInPlatform = getSettings().getBoolean(READ_WRITE_ACTIVATED, false);
     if (!readWriteActivatedInPlatform) {
       if (throwException) {
-        throw new SilverCrawlerForbiddenActionException(
-            "SilverCrawlerSessionController.checkRWSettingsAccess", SilverpeasException.ERROR,
+        throw new SilverCrawlerForbiddenActionException(CHECK_RWSETTINGS_ACCESS, SilverpeasException.ERROR,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "SilverpeasException"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)